pingdom-checks/direct-deposit-management.ping

```diff
@@ -0,0 +1,35 @@
+# -*- sh -*-
+
+DIRECT_DEPOSIT_MANAGEMENT_SLACK_ID="${SLACK_VASDVP_MONITORING}, ${SLACK_API_ALARMS_DIRECT_DEPOSIT_MANAGEMENT}"
+DIRECT_DEPOSIT_MANAGEMENT_BACKEND_SLACK_ID="${SLACK_LIGHTHOUSE_ALERTS}, ${SLACK_API_ALARMS_DIRECT_DEPOSIT_MANAGEMENT}"
+
+#
+# Sandbox
+#
+
+# Direct Deposit Management Health Check
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-direct-deposit-management-health \
+  -a host=sandbox-api.va.gov \
+  -a url="/services/direct-deposit-management/v1/healthcheck/liveness" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="direct-deposit-management" \
```

Comment:
@jkelly-va fyi, I renamed this from `direct-deposit` to match the full API name.